### PR TITLE
Update BlueShift.dmm

### PR DESCRIPTION
**Pull request by IsaacTheSharkWolf**
Fixed various ID and syntax issues, upgraded some hallway APCs to highcap ones, fixed their directions too (hopefully)

### CODE REVIEW

File: _maps/map_files/Blueshift/BlueShift.dmm [hunk 1]
```diff
@@ -98975,7 +99003,7 @@
 "oJb" = (
 /obj/effect/spawner/structure/window/reinforced,
 /obj/machinery/door/poddoor/preopen{
-	id = "Xeno11";
+	id = ''Xeno11'';
```
Comment (Gandalf2k15) [inline _maps/map_files/Blueshift/BlueShift.dmm]: Make sure you don't have any of these ''

Comment (IsaacTheSharkWolf) [reply _maps/map_files/Blueshift/BlueShift.dmm]: i tried to find that one but i could not find where the object in question was

Comment (IsaacTheSharkWolf) [reply _maps/map_files/Blueshift/BlueShift.dmm]: wait i can just use notepad++ probably

Comment (Gandalf2k15) [inline _maps/map_files/Blueshift/BlueShift.dmm]:
```suggestion
	id = "Xeno11";
```

**Comment by ORCACommander** [comment]
```
loading map_files/Blueshift/BlueShift.dmm
_maps/map_files/Blueshift/BlueShift.dmm:8550:error: invalid variable name: obj_integrity
_maps/map_files/Blueshift/BlueShift.dmm:9785:error: bad type or variable: /obj/item/melee/classic_baton/telescopic
_maps/map_files/Blueshift/BlueShift.dmm:9785:error: expected )
Failed to load map file /home/runner/work/Skyrat-tg/Skyrat-tg/_maps/map_files/Blueshift/BlueShift.dmm.
```

File: _maps/map_files/Blueshift/BlueShift.dmm (hunk 1 above)
Comment (Gandalf2k15) [inline _maps/map_files/Blueshift/BlueShift.dmm]:
```suggestion
	id = "Xeno11";
```

**Comment by IsaacTheSharkWolf** [comment]
> ```
> loading map_files/Blueshift/BlueShift.dmm
> _maps/map_files/Blueshift/BlueShift.dmm:8550:error: invalid variable name: obj_integrity
> _maps/map_files/Blueshift/BlueShift.dmm:9785:error: bad type or variable: /obj/item/melee/classic_baton/telescopic
> _maps/map_files/Blueshift/BlueShift.dmm:9785:error: expected )
> Failed to load map file /home/runner/work/Skyrat-tg/Skyrat-tg/_maps/map_files/Blueshift/BlueShift.dmm.
> ```

ok so the issue is those dumb windoors in the AI sat

File: _maps/map_files/Blueshift/BlueShift.dmm [hunk 2]
```diff
@@ -11320,8 +11322,8 @@
 /area/hallway/secondary/service)
 "bLN" = (
 /obj/machinery/door/poddoor/shutters/preopen{
-	id = "Xeno9";
-	name = "Xenobio Pens Containment"
+	id = ''Xeno9'';
```
Comment (ORCACommander) [inline _maps/map_files/Blueshift/BlueShift.dmm:11325]: use quotation marks

File: _maps/map_files/Blueshift/BlueShift.dmm [hunk 3]
```diff
@@ -8546,6 +8547,7 @@
 	icon_state = "rightsecure";
 	layer = 4.1;
 	name = "Secondary AI Core Access";
+	obj_integrity = 300;
```
Comment (ORCACommander) [inline _maps/map_files/Blueshift/BlueShift.dmm:8550]: delete this

File: _maps/map_files/Blueshift/BlueShift.dmm [hunk 4]
```diff
@@ -9780,7 +9782,7 @@
 "bza" = (
 /obj/structure/safe,
 /obj/item/storage/secure/briefcase{
-	contents = newlist(/obj/item/clothing/suit/armor/vest,/obj/item/gun/ballistic/automatic/pistol,/obj/item/suppressor,/obj/item/melee/baton/telescopic,/obj/item/clothing/mask/balaclava,/obj/item/bodybag,/obj/item/soap/nanotrasen)
+	contents = newlist(/obj/item/clothing/suit/armor/vest,/obj/item/gun/ballistic/automatic/pistol,/obj/item/suppressor,/obj/item/melee/classic_baton/telescopic,/obj/item/clothing/mask/balaclava,/obj/item/bodybag,/obj/item/soap/nanotrasen)
```
Comment (ORCACommander) [inline _maps/map_files/Blueshift/BlueShift.dmm]: this change breaks ci

**Review by ORCACommander — CHANGES_REQUESTED** [review]

**Review by ORCACommander — APPROVED** [review]

**Pull request merged**